.github/ISSUE_TEMPLATE/lifecycle_template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification creation/plan/release... lifecycle review
+title: ''
+labels: creation,plan,progress,release
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
+
+- [ ] Specification name and version
+- [ ] Add a label from one of: creation, plan, progress, release
+- [ ] A link to a directory under https://download.eclipse.org/microprofile/staging in the form orgeclipsemicroprofile-NNN where
+NNN is the staging repository id assigned to the staged with the following: (Does not apply to creation/plan review lifecycle events.)
+(micropfile-config, microprofile-health, ...) and x.y is the release major.minor version, and the directory contains the following.
+   - [ ] Specification PDF in the form of microprofile-_project_-spec.pdf where
```

Comment:
For Jakarta EE, we decided that the file names should have the version number as well since the files may get separated from the directory structure once they are downloaded.  So, `-spec-x.y.pdf` as an example.

Reply:
I checked all of MP specs. There are two types of the formats: spec.pdf and spec-x-y.pdf in a rough split ratio. Kevin made a good point. Let me double check the specs and fix/raise issues to get them fixed.